tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot

```diff
@@ -31,6 +31,15 @@ Test if all c8y services are down
     c8y-configuration-plugin
     c8y-log-plugin
 
+
+Test if all c8y services are using configured service type
+    [Template]     Check if a service using configured service type
```

Comment:
This test is probably okay but structure of this test looks problematic. This `Check if a service using configured service type` performs the `tedge config set` and `tedge reconnect` as part of this keyword and those commands will be called repeatedly for each service name argument that is passed to it(`tedge-mapper-c8y`, `tedge-agent` etc). That would unnecessarily call `tedge reconnect` for each service which is completely unnecessary, right? Why not do the `tedge config set` and `tedge reconnect` **once** as part of the test itself and then call the template that validates the service type for each input?

Reply:
The `tedge reconnect` is not needed here, just restarting the `tedge-mapper-c8y` is enough. I have updated the test. 

Since the `template` pattern is used here, there is a restriction to call the custom test settings before starting these tests at **once**.